### PR TITLE
Next steps - got a second line in the poem.

### PR DESCRIPTION
**Pull request by EmbeddedMan**
Please review my change.

### CODE REVIEW

File: git-poem.txt [hunk 1]
```diff
@@ -1,5 +1,3 @@
-there once was a man from minnetonka
-This would be a lot easier for me if it were code and not prose
-I can't think of anything that rhymes with Minnetonka
-It's going to be a lame ass poem
-How about "Willy Wonka"?
+There once was a man from Minnetonka
+His chocolates were better than Wonka
```
Comment (EmbeddedMan) [inline git-poem.txt]: I wanted "Wonka" to be "Wonka's" but I'm not sure the syllables are correct for a limerick.  What do you think @davidtcalabrese

Comment (davidtcalabrese) [reply git-poem.txt]: I think "Wonka's" works great. Limericks are frequently non-sensical so I wouldn't be concerned too much with breaking the rules. I think we should maintain the traditional _aabba_ rhyme scheme and try our best to match the anapestic trimeter. 

Unlike in programming, poems aren't broken when the syntax is changed slightly, and sometimes they are enhanced. Shakespeare was famous for writing in iambic pentameter but if you look closely at his plays you see he strays from it all the time! Some lines have added syllables, some are less than ten syllables. Sometimes he just wrote in prose! I had a theater professor who said that straying from this rigidity in drama makes it seem more natural.

Comment (EmbeddedMan) [reply git-poem.txt]: Excellent. I will update the PR to reflect this advice.

Comment (davidtcalabrese) [reply git-poem.txt]: So is this ready to be resolved and merged? What is the typical process for doing this?

Comment (EmbeddedMan) [reply git-poem.txt]: I have not made the proposed change yet. (it's still Wonka and not Wonka's) Once that change is made, then you can approve the PR and then merge it.

**Comment by EmbeddedMan** [comment]
Updated poem. Now it's complete, but I don't like the last line's duplicate use of the word 'just'.

**Review by EmbeddedMan — COMMENTED** [review]
I'm fine merging this version in, with future ideas welcome to make it better.

File: git-poem.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 There once was a man from Minnetonka
-His chocolates were better than Wonka
-
+His chocolates were better than Wonka's
+His molds were unique
+With just one small tweak
+They might just land him in Lubyanka
```
Comment (davidtcalabrese) [inline git-poem.txt:5]: Love it!

**Review by davidtcalabrese — APPROVED** [review]
We can revisit the limerick later now that we have something more fun going on. I'm sure you can consult Nancy for helpful hints.

**Pull request merged**